Sources/MongoSwift/Database.swift

```diff
@@ -96,10 +105,11 @@ public struct CreateCollectionOptions: Encodable {
         self.validationLevel = validationLevel
         self.validator = validator
         self.viewOn = viewOn
+        self.writeConcern = writeConcern
     }
 
-    // Encode everything except `readConcern`. We skip it because we don't actually
-    // send it with the initial command, we just set it on the collection after its creation.
+    // Encode everything except `readConcern` and 'writeConcern`. We skip them because we don't actually
+    // send them with the initial command, we just set them on the collection after its creation.
```

Comment:
I'm confused. Does the `createCollection()` helper return a MongoCollection object and _that_ is the only place these options are applied?

The [`create` command](https://docs.mongodb.com/manual/reference/command/create/) is a [command that writes](https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#other-commands-that-write) and supports a `writeConcern` option on its own, so I would expect that option to be used for executing the command.

Could we work around this by removing the notion of soliciting read and write concern objects just to apply them to the return value? I'd argue in favor of removing the syntactic sugar to have `createCollection` return a MongoCollection. This is going to get more ambiguous if we need to also take a read preference down the line ([SWIFT-31](https://jira.mongodb.org/browse/SWIFT-31)).

Reply:
yup, that's what I'm proposing. `createCollection` takes in a `WriteConcern` still, but we use it for the command rather than setting it on the collection. and it returns the newly created collection with inherited RC/WC/RP, and if the user wants to specify something else, they need to do
```swift
_ = try db.createCollection("coll1")
let coll1 = try db.collection("coll1", options: CollectionOptions(readConcern: ..., writeConcern: ...))
```

this is kinda weird, but I'm not sure how else we'd do it without breaking all of our tests and also probably breaking stitch code.

Reply:
That works for me.